docs/writing-stories/introduction.md

```diff
@@ -155,9 +155,9 @@ What’s more, you can import args to reuse when writing stories for other compo
 
 <!-- prettier-ignore-end -->
 
-When Button’s signature changes, you only need to change Button’s stories to reflect the new schema. ButtonGroup’s stories will automatically be updated. This pattern allows you to reuse your data definitions up and down your component hierarchy, making your stories more maintainable.
+When Button’s signature changes, you only need to change Button’s stories to reflect the new schema, and ButtonGroup’s stories will automatically be updated. This pattern allows you to reuse your data definitions across the component hierarchy, making your stories more maintainable.
 
-That’s not all! Each of the args from the story function are live editable using Storybook’s [controls](../essentials/controls.md) panel. This means your team can dynamically change components in Storybook to stress test and find edge cases.
```

Comment:
I've noticed lots of changes from "This..." to "It...", particularly at the start of a sentence or clause.

Would you mind sharing why you made those changes? They don't read as straightforwardly to me, but perhaps I'm missing something.

Reply:
@kylegach to give you some context the switch is to prevent unclear precedents. With the switch, it can help readers understand better the messaging a bit better.